.mergify.yml

```diff
@@ -34,6 +34,7 @@ pull_request_rules:
       - base=master
       - -label="needs changelog label"
       - -label=blocked
+      # - "#approved-reviews-by>=1"
```

Comment:
How about just removing this?

Reply:
@aldesantis sorry for the confusion, I was hoping to see the red mark, then uncomment this line, and see it gone for good. 
Unfortunately Mergify ignores PRs that are touching its configuration, anyway I was told this is the line we need to add by Mergify support staff, so 🤞